package.json

```diff
@@ -33,39 +28,27 @@
     "@babel/plugin-proposal-object-rest-spread": "7.8.3",
     "@babel/plugin-transform-react-jsx": "7.8.3",
     "@babel/polyfill": "7.8.3",
-    "@wordpress/api-fetch": "3.11.0",
     "@wordpress/babel-preset-default": "4.10.0",
-    "@wordpress/blob": "2.7.0",
     "@wordpress/block-editor": "3.7.1",
     "@wordpress/blocks": "6.9.0",
     "@wordpress/browserslist-config": "2.6.0",
     "@wordpress/components": "9.2.1",
     "@wordpress/compose": "3.11.0",
-    "@wordpress/core-data": "2.12.0",
     "@wordpress/data": "4.14.0",
-    "@wordpress/date": "3.8.0",
-    "@wordpress/dom": "2.8.0",
     "@wordpress/dom-ready": "2.7.0",
     "@wordpress/e2e-test-utils": "4.3.0",
     "@wordpress/edit-post": "3.13.1",
-    "@wordpress/editor": "9.12.1",
     "@wordpress/element": "2.11.0",
     "@wordpress/eslint-plugin": "3.4.1",
     "@wordpress/hooks": "2.7.0",
-    "@wordpress/html-entities": "2.6.0",
     "@wordpress/i18n": "3.9.0",
-    "@wordpress/is-shallow-equal": "1.8.0",
     "@wordpress/jest-puppeteer-axe": "1.6.0",
-    "@wordpress/keycodes": "2.9.0",
     "@wordpress/nux": "3.12.1",
     "@wordpress/plugins": "2.12.0",
     "@wordpress/postcss-themes": "2.3.0",
-    "@wordpress/rich-text": "3.12.0",
     "@wordpress/scripts": "5.1.0",
     "@wordpress/server-side-render": "1.8.1",
```

Comment:
Not needed anymore (was used for Latest Stories block)

Reply:
Also means removing the `assets/src/polyfills/wp-server-side-render.js` file and adapting `amp_register_default_scripts()` in PHP.